### PR TITLE
Fix: istio/gke version conflict by unpinning gke version

### PR DESCRIPTION
**Pull request by nicoleczhu**
I just checked today, issue #482 doesn't happen anymore with the GKE's [Oct 2 release](https://cloud.google.com/kubernetes-engine/docs/release-notes) where: "Version 1.16.13-gke.401 is the new default version for clusters with no & stable channels."

### CODE REVIEW

**Review by simonz130 — APPROVED** [review]

**Review by Daniel-Sanche — APPROVED** [review]

**Comment by nicoleczhu** [comment]
Oops, I need to fix something. Don't merge yet

**Comment by nicoleczhu** [comment]
Update: TIL GKE has a rollout schedule that only gets to central zones on day 4, i.e. today. Has nothing to do with Terraform...🤦

I removed the GKE pin again. 

Ready to merge once e2e tests pass

**Pull request merged**
